conf/nebula-storaged-listener.conf.default

```diff
@@ -0,0 +1,58 @@
+########## nebula-storaged-listener ###########
+########## basics ##########
+# Whether to run as a daemon process
+--daemonize=true
+# The file to host the process id
+--pid_file=pids_listener/nebula-storaged.pid
+# Whether to use the configuration obtained from the configuration file
+--local_config=true
+
+########## logging ##########
+# The directory to host logging files
+--log_dir=logs_listener
+# Log level, 0, 1, 2, 3 for INFO, WARNING, ERROR, FATAL respectively
+--minloglevel=0
+# Verbose log level, 1, 2, 3, 4, the higher of the level, the more verbose of the logging
+--v=0
+# Maximum seconds to buffer the log messages
+--logbufsecs=0
+# Whether to redirect stdout and stderr to separate output files
+--redirect_stdout=true
+# Destination filename of stdout and stderr, which will also reside in log_dir.
+--stdout_log_file=storaged-stdout.log
+--stderr_log_file=storaged-stderr.log
+# Copy log messages at or above this level to stderr in addition to logfiles. The numbers of severity levels INFO, WARNING, ERROR, and FATAL are 0, 1, 2, and 3, respectively.
+--stderrthreshold=2
+# Wether logging files' name contain timestamp.
+--timestamp_in_logfile_name=true
+
+########## networking ##########
+# Meta server address
+--meta_server_addrs=127.0.0.1:9559
+# Local ip
+--local_ip=127.0.0.1
+# Storage daemon listening port
+--port=9789
+# HTTP service ip
+--ws_ip=127.0.0.1
+# HTTP service port
+--ws_http_port=19789
+# heartbeat with meta service
+--heartbeat_interval_secs=10
+
+########## storage ##########
+# Listener wal directory. only one path is allowed.
+--listener_path=data/listener
+# This parameter can be ignored for compatibility. let's fill A default value of "data"
+--data_path=data
+# The type of part manager, [memory | meta]
+--part_man_type=memory
+# The default reserved bytes for one batch operation
+--rocksdb_batch_size=4096
+# The default block cache size used in BlockBasedTable.
+# The unit is MB.
+--rocksdb_block_cache=4
+# The type of storage engine, `rocksdb', `memory', etc.
+--engine_type=rocksdb
+# The type of part, `simple', `consensus'...
+--part_type=simple
```

Comment:
Could remove those useless configurations~

Reply:
It‘s copy from nebula-storage-listener.conf.production